pkgs/tools/misc/lipx/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, lib, fetchFromGitHub, python }:
+
+with lib;
+
+stdenv.mkDerivation rec {
+  name = "lipx-${version}";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "kylon";
+    repo = "Lipx";
+    rev = "1c01fcff06f1015750ad5461d5c898cba692326d";
```

Comment:
Please either move the version to ``unstable-${date}`` or use the commit rev of the "release", which in this case looks like ``e66dc12c6c22c060e5c24fdf2698d7e6c2543b7b`` for 1.2.?